web/styles/settings.css

```diff
@@ -656,11 +652,46 @@ input[type="checkbox"] {
     margin: 10px 0;
 }
 
-.add-new-default-stream-box {
-    & input[type="text"] {
-        padding: 6px;
+#add-default-stream-modal {
+    .dropdown-list-widget {
+        display: inline;
+    }
+
+    .dropdown-widget-button {
+        display: inline-flex;
+    }
+
+    .dropdown-menu {
+        position: fixed;
+        top: 80px;
+        left: 26px;
+    }
+
+    .default_stream_choices_table {
+        margin-top: 2px;
+    }
+
+    .choice-row {
+        margin-bottom: 10px;
+
+        &:last-of-type {
+            .delete-choice {
+                display: none;
+            }
+        }
+    }
+}
+
+.add_default_streams_button_container {
+    float: right;
+    margin-bottom: 6px;
```

Comment:
I think this `margin-bottom` property is not needed.

Reply:
Removed. Thanks.